.travis.yml

```diff
@@ -10,6 +10,7 @@ services: mongodb
 
 install:
   - pip install tornado
+  - pip install pymongo[encryption]
```

Comment:
I think this should be restricted to PyMongo<4 
```
pip install pymongo[encryption]>=3.11,<4
```

Reply:
What about `pip install -e '.[encryption]'`?

Reply:
sounds good!

Reply:
will go with shane's suggestion here to keep all the version info in setup.py